backend/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import { Card } from "./utils/card";
+import express, { Request, Response } from "express";
+import cors from "cors";
+
+const app = express();
+const port = 3000;
+
+const cards: Card[] = [
+  {
+    id: 1,
+    term: "Git was created by Linus Torvalds",
+    definition: "True",
+  },
+  {
+    id: 2,
+    term: "HTTP stands for HyperText Transfer Protocol",
+    definition: "True",
+  },
+  {
+    id: 3,
+    term: "HTTP is a secure protocol",
+    definition: "False",
+  },
+  {
+    id: 4,
+    term: "Computer Science is the best major",
+    definition: "True",
+  },
+];
+
+const deck = { cards: cards };
+
+app.use(cors());
+app.use(express.json());
+
+app.get("/", (_: Request, res: Response) => {
```

Comment:
You _could_ put this all in sep. files as bEsT pRaCtIcE but yeah this all looks good